### PR TITLE
Fix/dark theme #189

### PR DESCRIPTION
**Pull request by j-chimienti**
screen shot 2018-07-27 at 8 44 13 am
screen shot 2018-07-27 at 8 44 06 am

### CODE REVIEW

**Comment by j-chimienti** [comment]
issues:

theme not saving on mobile

changes:

font color = white
hyperlink color = white / blue

File: app/scripts/directives/cssThemeDrtv.html [hunk 1]
```diff
@@ -41,23 +42,50 @@
...
+}
+
+a.isActive {
+
+    color: white;
+}
+
+.form-control {
+
+    color: #333;
```
Comment (yograterol) [inline app/scripts/directives/cssThemeDrtv.html]: Just like comment, why not to have all colors in rgb/a and not mixed hex, rbg/a, word, what do you think?

Comment (j-chimienti) [reply app/scripts/directives/cssThemeDrtv.html]: I like your style

**Comment by j-chimienti** [comment]
screen shot 2018-07-27 at 12 50 03 pm
screen shot 2018-07-27 at 12 49 54 pm
screen shot 2018-07-27 at 12 49 33 pm

**Pull request merged**